module_4/framework/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:8.1-cli
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    unzip \
+    && docker-php-ext-install pdo pdo_mysql
+
+# Install Composer
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
```

Comment:
@Hlopec If it was a production Dockerfile, it would lack a version freeze for the composer.